src/test/components/__snapshots__/MarkerChart.test.js.snap

```diff
@@ -369,6 +369,16 @@ Array [
             "rgba(255, 255, 255, 0.0)",
           ],
         ],
+        "results": Array [
+          Object {
+            "isThrow": false,
+            "value": undefined,
+          },
+          Object {
+            "isThrow": false,
+            "value": undefined,
+          },
+        ],
```

Comment:
mock functions now track results in addition to parameters, that's why we see it in the serialized version too.

Looking at this snapshot, it looks like that `addColorStop` could be just a `jest.fn()` instead of a full `spyLog` in https://github.com/devtools-html/perf.html/blob/87eb4c73b3e75eea24c82e8f7950142402bd905b/src/test/fixtures/mocks/canvas-context.js#L38-L40
Indeed the data seems to be duplicated: we see both the calls to `addColorStop` above and the mock serialization.

What do you think ?

Reply:
I'm fine however you want to do it.